src/access/AccessControl.cpp

```diff
@@ -88,6 +159,15 @@ CHIP_ERROR AccessControl::Check(const SubjectDescriptor & subjectDescriptor, con
     // Don't check if using default delegate (e.g. test code that isn't testing access control)
     ReturnErrorCodeIf(&mDelegate == &mDefaultDelegate, CHIP_NO_ERROR);
 
+    {
+        char buf[64]; // fits 5 cats
+        ChipLogDetail(DataManagement,
+                      "AccessControl: checking f=%" PRIu8 " a=%c s=0x%" PRIx64 " t=%s c=0x%" PRIx32 " e=%" PRIu16 " p=%c",
```

Comment:
PRIx64 blows up on a bunch of the embedded libcs.  Please use ChipLogFormatX64/ChipLogValueX64.

Reply:
Done, more verbose though. :-(